skbio/stats/ordination/__init__.py

```diff
@@ -113,12 +113,13 @@
 from ._correspondence_analysis import CA
 from ._redundancy_analysis import RDA
 from ._canonical_correspondence_analysis import CCA
-from ._principal_coordinate_analysis import PCoA
+from ._principal_coordinate_analysis import PCoA, PCoABase
```

Comment:
:+1: Base classes probably don't make sense to export generally, unless you would expect users to extend said base classes.